app/views/organizations/adoptable_pets/show.html.erb

```diff
@@ -1,3 +1,5 @@
+<%= stylesheet_link_tag "https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.5/font/bootstrap-icons.css", rel: "stylesheet" %>
```

Comment:
We should not need any CDN, or custom scripts.
Our app is using a theme - see the issue where I link to it. Check that example I provided for a tooltip - you should be able to use the same data attributes and classes to make this tooltip work. 

Further, this tooltip should only show when we display the 'awaiting data' status...it doesn't make sense to show it for any other status, because the tooltip is specific to that status.

Reply:
I just read through the Bootstrap 5 documentation and it seems we'll still have to add a script to initialize the tooltip since they're opt-in for performance reasons, right? [Bootstrap Documentation](https://getbootstrap.com/docs/5.0/components/tooltips/)

I read that some themes will auto-initialize but I'm not sure if this theme does that since I can't find any a `main.js` or `theme.js` or any other file that contains similar code.

Reply:
Try the theme implementation. The JS is bundled with it and other JS featues work from it.

Reply:
Yeah I tried

```
<button type="button" class="btn btn-secondary" data-bs-toggle="tooltip" data-placement="top" title="Tooltip on top">
  Tooltip on top
</button>
```

but it doesn't seem to pop up without the script.

https://github.com/user-attachments/assets/c17cf9ac-9dc3-4223-8562-d7d260580b04